docs/docs/workflows/settings/README.md

```diff
@@ -50,6 +50,23 @@ On any workflow with auto-retry enabled, you can optionally choose to **Send not
 
 For custom control over error handling, you can implement error logic in code steps (e.g. `try` / `catch` statements in Node.js code), or [create your own custom error workflow](/workflows/errors/#process-errors-with-custom-logic-instead-of-email).
 
+## Data Retention Controls
+By default, Pipedream logs and surfaces event data from your deployed workflow in two places in the Pipedream UI, so you can easily understand the state of your workflow and debug any errors that pop up:
```

Comment:
Consider separating the storage of data from the places where you can view events.

> By default, Pipedream stores exports, logs, and other data tied to workflow executions. You can view these logs in two places: